src/index.js

```diff
@@ -28,7 +28,7 @@ function openregisterLocationPicker (opts) {
   opts.minLength = opts.minLength || 2
 
   opts.onSelect = opts.onSelect || ((result) => {
-    var requestedOption = Array.prototype.filter.call(opts.selectElement.options, o => o.innerHTML === result && result.name)[0]
+    var requestedOption = Array.prototype.filter.call(opts.selectElement.options, o.innerText === (result && result.name))[0]
```

Comment:
This is meant to be an [ES6 arrow function](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions), so you should keep the arrow in, like:

```js
o => o.innerText === (result && result.name)
```